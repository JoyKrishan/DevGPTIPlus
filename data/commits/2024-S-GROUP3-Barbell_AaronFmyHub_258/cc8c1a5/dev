
scripts/json/dp3/*.json