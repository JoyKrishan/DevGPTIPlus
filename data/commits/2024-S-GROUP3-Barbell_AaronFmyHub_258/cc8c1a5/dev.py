API_TOKEN = os.environ['API_TOKEN']
    print(f'fetching data from url: {url}')
issues_json = []
for page in range(1,10):
    url = f'https://api.github.com/repos/UNLV-CS472-672/2024-S-GROUP3-Barbell/issues?state=all&per_page=100&page={page}'
    issues_json += get_gh_json(url)
comments_json = []
for page in range(1,10):
    url = f'https://api.github.com/repos/UNLV-CS472-672/2024-S-GROUP3-Barbell/issues/comments?per_page=100&page={page}'
    comments_json += get_gh_json(url)
pulls_json = []
for page in range(1,10):
    url = f'https://api.github.com/repos/UNLV-CS472-672/2024-S-GROUP3-Barbell/pulls?state=all&per_page=100&page={page}'
    pulls_json += get_gh_json(url)
reviews_json = []
for idx, pr in enumerate(pulls_json):
    try:
        pr_num = pr['number']
        url = f'https://api.github.com/repos/UNLV-CS472-672/2024-S-GROUP3-Barbell/pulls/{pr_num}/reviews'
        print(f'{idx}: fetching reviews for pr #{pr_num}')
        reviews_json += get_gh_json(url)
        time.sleep(1)
    except TypeError as e:
        print(e)


with open('json/dp3/issues.json', 'w') as f:
    json.dump(issues_json, f, indent=4)

with open('json/dp3/comments.json', 'w') as f:
    json.dump(comments_json, f, indent=4)

with open('json/dp3/pulls.json', 'w') as f:
    json.dump(pulls_json, f, indent=4)

with open('json/dp3/reviews.json', 'w') as f:
    json.dump(reviews_json, f, indent=4)