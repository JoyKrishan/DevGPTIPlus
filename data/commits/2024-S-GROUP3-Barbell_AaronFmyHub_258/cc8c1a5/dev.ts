import chat from '../mock-data/chat.json'
import exercise from '../mock-data/exercise.json'
import message from '../mock-data/message.json'
import users from '../mock-data/user.json'
import workoutLog from '../mock-data/workoutLog.json'
import workoutTemplate from '../mock-data/workoutTemplate.json'
    /* */