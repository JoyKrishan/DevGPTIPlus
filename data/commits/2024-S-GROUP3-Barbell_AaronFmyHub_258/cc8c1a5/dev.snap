    className="placeholder:color-[#717171] px-3 py-[6px] text-[16px] text-white"